packages/next/src/server/config-shared.ts

```diff
@@ -289,6 +282,13 @@ export interface ExperimentalConfig {
 
   webVitalsAttribution?: Array<(typeof WEB_VITALS)[number]>
 
+  /**
+   * A list of extensions to resolve when importing files.
+   *
+   * @see [Resolve Extensions](https://nextjs.org/docs/app/api-reference/next-config-js/turbo#resolve-extensions)
```

Comment:
These docs live here: https://github.com/vercel/next.js/blob/669712cbf97f3de273cd8cf5460ee4bb17f2f16e/docs/02-app/02-api-reference/05-next-config-js/turbo.mdx#resolve-extensions

You should document both where it was in nextjs v14.0 (`experimental.turbo.resolveExtensions`) and where it is in v15.0 (experimental.resolveExtensions), as well as the change in behavior (also affects webpack in v15.0).